server/config.go

```diff
@@ -36,6 +36,8 @@ type Repository struct {
 	InstanceSetupScript        string
 	InstanceSetupUpgradeScript string
 	JobName                    string
+	GreetingTeam               string // Team of staff who is in charge of managing new community PRs for this repo
```

Comment:
```suggestion
        // GreetingTeam is the GitHub team responsible for triaging non-member PRs for this repo.
	GreetingTeam               string 
```

Reply:
Proposing `non-member` to highlight the distinction being made in not applying this process to members of the repository.